### PR TITLE
Fix bug with negative numbers

### PR DESCRIPTION
**Pull request by ruipneves**
I was looking for Math.round() documentation when I came across this bug. After concluding that roundTo was not behaving as expected I tried to fix the problem. The problem was that with negative numbers, Math.round() would go to the hight number, thus, not producing the expected results.

To solve this I checked if the requested function was the "round" function and if this was verified then the function would be applied to the absolute value of the number to round and then multiplied by it's original sign.

Is this the correct way to do it?
Any feedback is appreciated.

Fixes #134

### CODE REVIEW

**Comment by satazor** [comment]
@sindresorhus Should we remove support for node < 4 I guess?

File: package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
     "increment"
   ],
   "dependencies": {
-    "number-is-integer": "^1.0.0"
+    "number-is-integer": "^1.0.1"
```
Comment (sindresorhus) [inline package.json]: Don't do unrelated changes. This change also makes no difference.

**Comment by sindresorhus** [comment]
> Should we remove support for node < 4 I guess?

Same answer as https://github.com/sindresorhus/to-single-quotes/pull/14#issuecomment-274359445.

**Comment by sindresorhus** [comment]
@ruipneves Can you give the pull request a more descriptive title. Also, always link to the issue you're fixing using `Fixes #134`. That way it will close the issue when the pull request is merged.

File: index.js [hunk 2]
```diff
@@ -14,6 +14,9 @@ function round(fn, x, precision) {
 	var exponentNeg = precision > 0 ? 'e-' : 'e';
 	precision = Math.abs(precision);
 
+	if (fn === 'round') {
+		return Number(Math.sign(x) * (Math[fn](Math.abs(x) + exponent + precision) + exponentNeg + precision));
```
Comment (sindresorhus) [inline index.js]: `Math[fn]` => `Math.round`. We already know what the `fn` is.

**Comment by ruipneves** [comment]
@sindresorhus Thank you again for your feedback! The requested changes have been made.

**Comment by sindresorhus** [comment]
> Also, always link to the issue you're fixing using Fixes #134.

134 was just an example. That should be the actual issue number of what you're fixing. I'd recommend reading https://guides.github.com/features/mastering-markdown/ and https://github.com/tiimgreen/github-cheat-sheet

**Comment by sindresorhus** [comment]
Thank you @ruipneves :)
